src/commands/workers.rs

```diff
@@ -129,6 +140,52 @@ async fn create_worker(
     }))
 }
 
+/// Updates a worker with the specified ID.
+async fn update_worker(
+    chain_args: &ChainArgs,
+    path: Option<&Path>,
+) -> Result<Value, Box<dyn std::error::Error>> {
+    let worker: gevulot_rs::models::Worker = read_file(path).await?;
+    let mut client = connect_to_gevulot(chain_args).await?;
+    let me = client
+        .base_client
+        .write()
+        .await
+        .address
+        .clone()
+        .ok_or("No address found, did you set a mnemonic?")?;
+    let id = worker.metadata.id.ok_or("Worker ID not found")?;
+    client
+        .workers
+        .update(
+            MsgUpdateWorkerBuilder::default()
+                .creator(me)
```

Comment:
Just a question: should `update` command change _the creator_ of the worker?

Reply:
we need to set that field. In the cosmos-sdk this field is bound to the signer and is used to verify the message. We have a creator field on every message type that interacts with stuff that is owned by an account.